src/Enterspeed.Cli/Services/FileService/FilePathService.cs

```diff
@@ -0,0 +1,55 @@
+using Enterspeed.Cli.Domain.Models;
+
+namespace Enterspeed.Cli.Services.FileService;
+
+public class FilePathService : IFilePathService
+{
+    private const string SchemaDirectory = "schemas";
+
+    public string GetRelativeRootDirectoryPath()
+    {
+        return SchemaDirectory;
+    }
+
+    public string GetRootDirectoryPath()
+    {
+        return Path.Combine(Directory.GetCurrentDirectory(), SchemaDirectory);
+    }
+
+    private static string GetRelativeDirectoryPathBySchemaName(string schemaName)
+    {
+        return Path.GetDirectoryName(schemaName.TrimEnd(Path.DirectorySeparatorChar));
+    }
+
+    private static string GetDirectoryPathByFilePath(string filePath)
+    {
+        return Path.GetDirectoryName(filePath);
+    }
+
+    public string GetRelativeSchemaDirectoryPath(string currentSchemaFilePath)
+    {
+        var rootSchemaDirectoryPath = GetRootDirectoryPath();
+        var currentSchemaDirectoryPath = GetDirectoryPathByFilePath(currentSchemaFilePath);
+        var relativeSchemaDirectoryPath = currentSchemaDirectoryPath?.Replace(rootSchemaDirectoryPath, "");
+
+        if (!string.IsNullOrEmpty(relativeSchemaDirectoryPath) && relativeSchemaDirectoryPath.StartsWith(Path.DirectorySeparatorChar))
+        {
+            relativeSchemaDirectoryPath = relativeSchemaDirectoryPath.Remove(0, 1);
+        }
+
+        return relativeSchemaDirectoryPath;
+    }
+
+    public string GetAliasFromFilePath(string filePath)
+    {
+        return Path.GetFileNameWithoutExtension(filePath).Replace(".partial", "")
+            .Replace(".full", "");
+    }
```

Comment:
Should this be moved to the SchemaName Service, together with the `GetFileName` from the SchemaFileService? Then we don't have logic two places in regards to the schema type in the name?